packages/flutter/lib/src/rendering/copy_interceptor.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'selection.dart';
+
+/// Typedef for the inline intercept callback.
```

Comment:
nit: maybe this can include more information `Signature for the callback that allows the user to intercept copy operations`.

Reply:
Done